schema.ts

```diff
@@ -80,7 +89,7 @@ export const chatParticipants = pgTable(
     // authenticated users can read chat participant list
     crudPolicy({
       role: authenticatedRole,
-      read: sql`(select auth.user_id() in (select user_id from chat_participants where chat_id = ${table.chatId}))`,
+      read: false,
```

Comment:
Since we can't create a RLS policy for this rule, it's better to block its reading, and only use the view.